src/DynamoRevit/ViewModel/DynamoRevitViewModel.cs

```diff
@@ -33,7 +33,7 @@ public class DynamoRevitViewModel : DynamoViewModel
             RegisterWatch3DViewModel(watch3DVm, new DefaultRenderPackageFactory());
         }
 
-        public static DynamoRevitViewModel Start(StartConfiguration startConfiguration)
+        new public static DynamoRevitViewModel Start(StartConfiguration startConfiguration)
```

Comment:
why make this change, is there a base class version of this method you are hiding?

Reply:
@mjkkirschner Typically code I work with compiles without warning, when building Dynamo For Revit a slew of warnings come up, in a couple places the warning about "hiding" base implementation came up, so I added the new keyword to fix the warning.. I did not add this, it has always been there... so I presume it was just an ignored warning.